### ISSUE TITLE
Jester Selfkill Issue

### ISSUE THREAD

**Issue by dodorian88** [TTT-2/ttt2-role_jes#6]
Jester can kill himself per "kill" command in console and trigger his winstate. The Death Grip addon for TTT2 has the same problem.

**Comment by Alf21** [comment]
@saibotk could you fix this DeathGrip issue or is there an available hook?

**Comment by Histalek** [comment]
@saibotk any progress on this jester/deathgrip issue?

**Comment by saibotk** [comment]
I dont even know what the problem is.

**Comment by Histalek** [comment]
On a second thought i can only guess what exactly the deathgrip issue is.
but as the jester issue is most likely fixed i'm closing this issue.

Feel free to open up a new issue in this repository (if it is a jester specific issue) or open a new issue in the [deathgrip repository](https://github.com/TTT-2/ttt2-deathgrip) :)

**Comment by dodorian88** [comment]
@Histalek Thanks for the heads up and I'm sorry for not being specific back then. When one of the two connected deathgrip-players used the kill command, they could use that to kill the other one aswell.

TLDR: One writes "kill" in console, both death-grip players die.

But I'm not sure, if this problem still persists. If it does, gonna do what you said :)
